charts/prometheus-elasticsearch-exporter/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Elasticsearch stats exporter for Prometheus
 name: prometheus-elasticsearch-exporter
-version: 5.7.0
+version: 5.7.1
```

This change is not really a bug fix and as a result, the minor number should be raised - i.e. a new feature is being introduced.

```suggestion
version: 5.8.0
```

---

Updated!